From ffc5bf686da7db9d26dd19b43fc8a17d83bcc4d7 Mon Sep 17 00:00:00 2001
From: WANG Cong <xiyou.wangcong@gmail.com>
Date: Fri, 18 May 2012 13:38:17 +0800
Subject: [PATCH] check ifenslave instead of brctl in parse-bond.sh

This is obviously wrong, ifenslave instead of brctl is needed for bonding.

Cc: Harald Hoyer <harald@redhat.com>
Signed-off-by: WANG Cong <xiyou.wangcong@gmail.com>
---
 modules.d/40network/parse-bond.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/40network/parse-bond.sh b/modules.d/40network/parse-bond.sh
index 0a4c006..ae0ae97 100755
--- a/modules.d/40network/parse-bond.sh
+++ b/modules.d/40network/parse-bond.sh
@@ -18,7 +18,7 @@ if getarg bond= >/dev/null ; then
     if [ -z "$netroot" ] ; then
         die "No netboot configured, bond is invalid"
     fi
-    command -v brctl >/dev/null 2>&1 || die "No 'brctl' installed"
+    command -v ifenslave >/dev/null 2>&1 || die "No 'ifenslave' installed"
 fi
 
 # We translate list of slaves to space-separated here to mwke it easier to loop over them in ifup
